### DISCUSSION TITLE
Improving loading speed of audio

### DISCUSSION THREAD

**Discussion by pokemon963** [katspaugh/wavesurfer.js#3120] [Q&A]
Hi! I am using Wavesurfer to load the audio files of my website. On AWS, my audio files load under 1 minute. However, my professor wants us to use our school's server for the site in order to avoid payments since we can host on our school's server for free. But, due to the difference in performance of the servers, my school's server can take minutes to load the audio files. Are there any suggestions for how the loading can be better for my school's server? I did see stuff about pre-decoded peaks but was wondering if there is anything else that can be done?

**Comment by katspaugh** [comment]
Try downsizing the audio by converting it to a more efficient or lossy format.
